src/mail/view/MailViewer.js

```diff
@@ -847,6 +847,8 @@ export class MailViewer {
 					'undefined' !== typeof Array.from(sanitizeResult.html.querySelectorAll('*[style]'), e => e.style)
 					                            .find(s => s.color !== "" && s.color !== "inherit" && typeof s.color !== 'undefined')
 					|| 0 < Array.from(sanitizeResult.html.querySelectorAll('font[color]'), e => e.style).length
+					|| 0 < Array.from(sanitizeResult.html.querySelectorAll('*'), e => e.style)
```

Comment:
Could you mention this new condition in the comment above

Reply:
The comment was outdated anyways. I rephrased the code and comment, could you have a look again?